### ISSUE TITLE
ReadSVG: Edit SVG (advanced)

### ISSUE THREAD

**Issue by olear** [olear/openfx-arena#268]
Make it possible to edit the SVG source in Natron for advanced usage (PyPlugs etc).

**Comment by manuelsongokuh** [comment]
i need feedback you for SVG.. if i can?

**Comment by olear** [comment]
The edit feature is primarily for PyPlugs, where you can edit the raw XML and modify the image (simple color corrections etc etc) without launching Inkscape, it does **not** replace Inkscape.

**Comment by manuelsongokuh** [comment]
yes i dont replace inkscape, but i want to ask you about this issue:
when i open file SVG with node reader, you can add little feature:
change scale 200% will be re-rasterized from reader..? i know natron is not vector but when open file svg before setting 200% scale so will be rasterized result image is 200% without bad pixel..
for this i ask you..if possible?

example for actual read use 100% scale rasterized in pixel result normal, if i want change scale to 200% or 400% so will be convert in raster in 400% without bad pixel (not effect scale from pixel bu from vector scale to rasterized.. 

sorry for my english is not strong.. i try explain again..

**Comment by manuelsongokuh** [comment]
maybe i find information (but not sure, tou can go to see..)
http://imagemagick.org/Usage/formats/#png_density

**Comment by manuelsongokuh** [comment]
http://stackoverflow.com/questions/9853325/how-to-convert-a-svg-to-a-png-with-image-magick

**Comment by olear** [comment]
ReadSVG uses DPI, 90=100%

**Comment by manuelsongokuh** [comment]
ok, maybe not work correct for my viewer because setting of project is 1920x1080.. strange..
in viewer not see project 1920x1080..where is my wrong..?

[strange.ntp.zip](https://github.com/olear/openfx-arena/files/617575/strange.ntp.zip)

**Comment by olear** [comment]
The zip only contain the project file, so I'm unable to check anything.

**Comment by manuelsongokuh** [comment]
sorry for retardation reply:
when you open ntp and go to change dpi from 90 to 200, result from viewer is black out size..this is strange or error human?
